index.js

```diff
@@ -17,13 +17,22 @@
 import { Font } from "./src/lib-font/lib-font-wrapper";
 import Fondue from "./src/fondue/Fondue.js";
 
-export function fromPath(fontPath) {
+export async function fromPath(fontPath) {
+	const { readFile } = await import("node:fs/promises");
+
 	return new Promise((resolve, reject) => {
 		const font = new Font(fontPath);
 		font.onload = () => resolve(new Fondue(font));
 		font.onerror = (e) => reject(e.detail.message);
 
-		font.src = fontPath;
+		(async () => {
+			try {
+				const file = await readFile(fontPath);
+				font.fromDataBuffer(file.buffer, fontPath).catch(reject);
```

Comment:
The `catch` shouldn't be required here as it will be catched by the try-catch construct.

Reply:
Oh, a left-over that didn't seem to do any harm. Will remove if it's superfluous.